### DISCUSSION TITLE
Playerbots module - Got error Table 'acore_world.charsections_dbc' doesn't exist  when starting world server

### DISCUSSION THREAD

**Discussion by peckwood** [azerothcore/azerothcore-wotlk#15439] [Modules]
When starting world server, it reports

```
[1146] Table 'acore_world.charsections_dbc' doesn't exist
Your database structure is not up to date. Please make sure you've executed all queries in the sql/updates folders.
```

However, I cannot find table charsections_dbc in any of the sql files inside sql/updates.

I built the server from https://github.com/ZhengPeiRu21/azerothcore-wotlk/tree/Playerbot

**Comment by deadcorpseeater** [comment]
I'm so sorry for offtop, but where exactly sql/updates should be for updating database?

**Reply by peckwood** [reply]
In the data folder of your source code, I guess. like SOURCE_CODE_FOLDER/data/sql/updates

**Reply by deadcorpseeater** [reply]
> In the data folder of your source code, I guess. like SOURCE_CODE_FOLDER/data/sql/updates

do you mean the folder I compiled? not the compiled result?

**Reply by peckwood** [reply]
Not the compiled result, I don't see sql updates folder there

**Comment by fooxytv** [comment]
I am also getting the same error when trying to build from https://github.com/ZhengPeiRu21/azerothcore-wotlk/tree/Playerbot

Did you manage to resolve the error for yourself?

**Comment by Jzundai** [comment]
I am also getting the same error when trying to build from https://github.com/ZhengPeiRu21/azerothcore-wotlk/tree/Playerbot

**Comment by Sittlon** [comment]
@Jzundai @fooxytv @peckwood 
Seems that that sql file isnt included in the branch, but i found it in the module repo
https://github.com/ZhengPeiRu21/mod-playerbots/blob/AzerothCore/sql/world/world_charsections_dbc.sql

**Reply by MohamedMahmoud299** [reply]
exactly you gotta compile the module for that error to get fixed

**Reply by dreamingblackcat** [reply]
HI @Sittlon How do we compile the module? I've cloned the `mod-playerbots` into `azerothcore-wotlk/modules` directory and then recompile azerothcore-wotlk by running `cd build; make -j 8; make install`

But, it's still the same error. 

Or do we need to compile `mod-playerbots` separately?

**Reply by dreamingblackcat** [reply]
UPDATE: I got it working by completely recompiling the azerothcore. 
```
cd build
make -j 8; // This will rebuild azerothcore with mod-playerbot module
make install;
```
Once installation is done, run `worldserver` command and it will automatically load the sqls from the module.

**Comment by xXRaptorSc0pezXx** [comment]
To fix the problem just go to HeidiSQL --> File --> Run SQL file... --> navigate to \azerothcore-wotlk\modules\mod-playerbots\mod-playerbots\data\sql\world\base and manually run each of the files in there to create the tables since it doesn't do it automatically for some reason.

If you are trying to install the Azerothcore Playerbots branch without the Playerbots module added to the Azerothcore folder you will need to clone that repository first (clone it to \azerothcore-wotlk\modules\mod-playerbots\ keep in mind you will need to create the "mod-playerbots folder").
